lib/concurrent/thread_safe/util/striped64.rb

```diff
@@ -83,7 +83,7 @@ class Cell < Concurrent::AtomicReference
 
           # TODO: this only adds padding after the :value slot, need to find a way to add padding before the slot
           # @!visibility private
-          attr_reader *(Array.new(12).map {|i| :"padding_#{i}"})
+          attr_reader *(12.times.collect{ |i| "padding_#{i}".to_sym })
```

Comment:
Thank you for answering so quickly! I'm going to cherry-pick line directly to master.